namespaces/live.cloud-platform.service.justice.gov.uk/visit-someone-in-prison-backend-svc-dev/10-certificate.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-prison-visits-testing-helper-api-cert
+  namespace: visit-someone-in-prison-backend-svc-dev
+spec:
+  secretName: hmpps-prison-visits-testing-helper-api-cert
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+    - hmpps-prison-visits-testing-helper-api-dev.prison.service.justice.gov.uk
```

Comment:
There is a character limit on the common name field. https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/custom-domain-cert.html#obtaining-a-certificate. Can you either reduce the characters or add a short name followed y this long name as explained in the guide

Reply:
have added a shorter version,.. cheers